### DISCUSSION TITLE
Synchronizing the events in the app (iOS Cordova)

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#9695] [CLI - Capacitor / Cordova modes] [answered]
Currently I use Web Audio API for the synchronization of animation & sounds in my app (iOS Cordova) but on PROD build, everything is all around the place. Interestingly this did not pop up during the DEV mode. 

Any other alternatives to sync the events in the app?

**Comment by ontwikkelfabriek** [comment, marked as answer]
@i-mage 

Just curious, but why do you use a `Web` (Audio) Api when you're creating a `native` Cordova app?  Why not use a Cordova/Capacitor audio plugin that has much tighter integration with IOS compared to a Web Api?

btw I found these cordova/capacitor audio plugins:
https://github.com/capacitor-community/native-audio
https://www.createjs.com/docs/soundjs/classes/CordovaAudioPlugin.html
https://www.npmjs.com/package/cordova-plugin-audioinput

**Reply by i-mage** [reply]
As mentioned in the other post, the issue has been solved (https://quasar.dev/quasar-cli/developing-cordova-apps/preparation#switching-to-ios-wkwebview)

However, I will definitely have a look at the links you posted and eventually refactor the code. Thank you!
